packages/core/saved-objects/core-saved-objects-server-internal/src/saved_objects_service.ts

```diff
@@ -223,7 +224,8 @@ export class SavedObjectsService
     const waitForMigrationCompletion = node.roles.backgroundTasks && !node.roles.ui;
     const migrator = this.createMigrator(
       this.config.migration,
-      elasticsearch.client.asInternalUser,
+      // override the default Client settings
+      elasticsearch.client.asInternalUser.child(MIGRATION_CLIENT_OPTIONS),
```

Comment:
We're creating a new child for every migrator, I think we should rather create one child/client for all migrators/indices

Reply:
Ouch, good catch!